shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -226,14 +255,15 @@ public Options getOptions() {
    * @param sortField sort the result by this field
    */
   private void ls(AlluxioURI path, boolean recursive, boolean forceLoadMetadata, boolean dirAsFile,
-                  boolean hSize, boolean pinnedOnly, String sortField, boolean reverse)
+                  boolean hSize, boolean pinnedOnly, String sortField, boolean reverse,
```

Comment:
nit: is the indent correct?

Reply:
Checkstyles doesn't seem to complain about it.